packages/edit-site/src/components/welcome-guide/styles.js

```diff
@@ -119,7 +119,7 @@ export default function WelcomeGuideStyles() {
 								) }
 								<ExternalLink
 									href={ __(
-										'https://wordpress.org/support/article/styles-overview/'
+										'https://wordpress.org/documentation/article/wordpress-block-editor/'
```

Comment:
Thanks for working on this.

This doesn't seem to be the right link that has been updated. There are a few welcome guides, this one is for globals styles and has a different URL.

I see it still has a redirect though. The URL now resolves to https://wordpress.org/documentation/article/styles-overview/, so it might still be worth updating it, but also updating the other ones too.

There are a few that I found that have the link you mentioned:
- [Post editor welcome guide](https://github.com/WordPress/gutenberg/blob/012c4ef5bee63eb988079a38a6b24769ea021362/packages/edit-post/src/components/welcome-guide/default.js#L113)
- [Widget editor welcome guide](https://github.com/WordPress/gutenberg/blob/68d492a15be91cf621e91d5aa16d916ec7c1bd44/packages/edit-widgets/src/components/welcome-guide/index.js#L180)
- [Customize widgets welcome guide](https://github.com/WordPress/gutenberg/blob/709e2947a4653e5ce28967e5e78946a88931ef5b/packages/customize-widgets/src/components/welcome-guide/index.js#L73)

Reply:
Thank you for your comment. Yes you are correct, I have edited the wrong link in this case. I will recheck and edit the PR.

Reply:
> Thanks for working on this.
> 
> This doesn't seem to be the right link that has been updated. There are a few welcome guides, this one is for globals styles and has a different URL.
> 
> I see it still has a redirect though. The URL now resolves to https://wordpress.org/documentation/article/styles-overview/, so it might still be worth updating it, but also updating the other ones too.
> 
> There are a few that I found that have the link you mentioned:
> 
> * [Post editor welcome guide](https://github.com/WordPress/gutenberg/blob/012c4ef5bee63eb988079a38a6b24769ea021362/packages/edit-post/src/components/welcome-guide/default.js#L113)
> * [Widget editor welcome guide](https://github.com/WordPress/gutenberg/blob/68d492a15be91cf621e91d5aa16d916ec7c1bd44/packages/edit-widgets/src/components/welcome-guide/index.js#L180)
> * [Customize widgets welcome guide](https://github.com/WordPress/gutenberg/blob/709e2947a4653e5ce28967e5e78946a88931ef5b/packages/customize-widgets/src/components/welcome-guide/index.js#L73)

Thank you @talldan. I have adjusted the links for all suggested, as there is no reason to display the redirected url. This now fixes the problem I was originally addressing. 

For the initial url which I edited incorrectly, I have reverted this change for now, and will address it in a different PR to avoid confusion.